bombbot.py

```diff
@@ -212,8 +212,12 @@ def cancel_bomb(bot, trigger):
     """
     target = trigger.group(3) or None
     if not target:
-        bot.reply(STRINGS['CANCEL_WHOM'])
-        return
+        for bomb in BOMBS:
+            if trigger.nick == BOMBS[bomb]['bomber']:
+                target = BOMBS[bomb]['target']
+                break
+        if not target:
```

Comment:
Do I like this nested, identical `if`? No.

Does it do the job? Yeah…unfortunately.

See also commentary on parent `if`.